nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -452,6 +452,27 @@ The LdapUserGroupProvider has the following properties:
 
 NOTE: Any identity mapping rules specified in _nifi.properties_ will also be applied to the user identities. Group names are not mapped.
 
+==== ShellUserGroupProvider
+
+The ShellUserGroupProvider fetches user and group details from Unix-like systems using shell commands.
+
+This provider executes various shell pipelines with commands such as `getent` on Linux and `dscl` on MacOS.
+
+Supported systems may be configured to retrieve users and groups from an external source, such as LDAP or NIS.  In these cases the shell commands
+will return those external users and groups.  This provides administrators another mechanism to integrate user and group directory services.
+
+The ShellUserGroupProvider has the following properties:
+
+[options="header,footer"]
+|==================================================================================================================================================
+| Property Name | Description
+|`Initial Refresh Delay` | Number of seconds to delay the initial user/group listing refresh.
+|`Refresh Delay` | Number of seconds to pause between each listing refresh.
+|==================================================================================================================================================
+
+Like LdapUserGroupProvider, the ShellUserGroupProvider is commented out in the _nifi.properties_ file.  Refer to that comment for usage examples.
```

Comment:
Should be _authorizers.xml_ file not _nifi.properties_

Reply:
Good catch.  Updated ref and added the properties to the example.